collie/model/base/trainer.py

```diff
@@ -220,6 +232,18 @@ def __init__(self,
         elif verbosity is False:
             verbosity = 0
 
+        if weights_summary is not None:
+            warnings.warn('``weights_summary`` is deprecated and was replaced with ``max_depth``.',
+                          DeprecationWarning)
+
+        if terminate_on_nan is not None:
+            warnings.warn(
+                '``terminate_on_nan`` is deprecated and was replaced with ``detect_anomaly``.',
+                DeprecationWarning
+            )
+            if not detect_anomaly:
```

Comment:
I prefer being a bit more explicit here: 

```suggestion
            if detect_anomaly is None:
```

Reply:
`detect_anomaly` will never be `None` the default is `False` so I changed this line to `if detect_anomaly is False:` let me know what you think.